HW03/homework_3.py

```diff
@@ -0,0 +1,20 @@
+print("I love Python" * 42) #1
```

Comment:
This file shouldn't be here. Remove it, please

Reply:
directory and file are removed